test/e2e/test_runner.go

```diff
@@ -107,6 +113,17 @@ func contains(features []common.Feature, feature common.Feature) bool {
 	return false
 }
 
+// Get the actual typemeta of the Channel type.
+// TODO(Fredy-Z): This function is a workaround when there are both provisioner and Channel CRD in this repo.
+//                It needs to be removed when the provisioner implementation is removed.
+func getChannelTypeMeta(provisioner string, isCRD bool) *metav1.TypeMeta {
+	channelTypeMeta := common.ChannelTypeMeta
+	if isCRD {
+		channelTypeMeta = common.ProvisionerChannelMap[provisioner]
```

Comment:
sounded confusing to have the map called ProvisionerChannelMap. The channel CRDs are the Operator model. Maybe OperatorChannelMap ?

Reply:
Yeah it sounds better. Done.